docs/core-concepts/run.mdx

```diff
@@ -7,48 +7,107 @@ toc_min_heading_level: 2
 ---
 
-All UDFs have a secure HTTP endpoint callable via a [web request](/core-concepts/run/#http-requests) or [`fused.run`](/core-concepts/run/#fusedrun) that is authorized with a [shared](/core-concepts/run/#shared-token) or [private](/core-concepts/run/#private-token) token. The endpoint's response may be [cached](/core-concepts/run/#caching-responses) and have its type [specified](/core-concepts/run/#response-data-types).
+Fused UDF functions really shine once you start calling them from anywhere. You can do so in 2 main ways:
+1.  `fused.run()` in Python. All you need is [`fused` Python](/python-sdk/) package is installed
+    - This is when you want to use a UDF in another pipeline
+2.  [**HTTP call**](/core-concepts/run/#http-requests) from *anywhere*
+    - This is when you want to call a UDF outside of Python. For example receiving a [dataframe into Google Sheets](/user-guide/out/googlesheets/) or plotting points and images in a [Felt map](/user-guide/out/felt/)
 
 ## `fused.run`
 
-The first argument of `fused.run` specifies the UDF. [Reserved parameters](/core-concepts/write/#reserved-parameters) specify `bbox` and control how the UDF will be executed. Any additional arguments are passed to the function. The UDF runs as [Tile](/core-concepts/filetile/#call-fusedrun) if a [`bbox` is specified](/core-concepts/filetile/#call-fusedrun).
+All you need to run a UDF within Python is:
+- [`fused` package](/python-sdk/#install) installed
+- Access to your team projects to run private UDFs
 
-### Ways to run a UDF
+The simplest way to call a UDF is using a [public UDF name](/core-concepts/run/#public-udf-name) and simply calling it by `UDF_` + name. Let's take this simple UDF that returns [the location of the Eiffel Tower](https://github.com/fusedio/udfs/tree/main/public/Single_point_Eiffel_Tower) in a `GeoDataFrame`:
 
-You can specify the UDF to run in the first `udf` parameter of `fused.run` by name, tokens, or a UDF object.
+```python showLineNumbers
+import fused
+fused.run("UDF_Single_point_Eiffel_Tower")
+```
+
+import ImgSimpleETUDF from '@site/static/img/core-concepts/run-udfs/running_simple_eiffel_tower_udf.png';
+
+<div style={{textAlign: 'center'}}>
+<img src={ImgSimpleETUDF} alt="Simple UDF fused.run() returning a geodataframe" style={{width: 400}} />
+</div>
+
+There are a few other ways to run a UDF:
+- [By name from your account](/core-concepts/run/#name-from-your-account)
+- [By public UDF name](/core-concepts/run/#public-udf-name)
+- [Using a token](/core-concepts/run/#token)
+- [Using a `udf` object](/core-concepts/run/#udf-object)
+- [From Github URL](/core-concepts/run/#github-url)
+- [From git commit hash (most recommended for teams)](/core-concepts/run/#git-commit-hash-recommended-for-teams)
+
+### _Name_ (from your account)
+
+_When to use: When calling a UDF you made, from your own account._
+
+You can call any UDFs you have made simply by calling them by the name you have given your UDF when saving it
```

Comment:
So in this case you don't have to add the `UDF_`? Is that specific to "public" ones?

Reply:
That's right:
- public UDF -> Need to prefix with `UDF_`
- your own UDFs -> Call as is